README.md

```diff
@@ -13,32 +13,37 @@ be in its own folder inside the current one.
 ## Commands
 
 - `generate` / `g` `[project name]` - This command starts the process of generating your Node.js project.
-  You will have to answer a couple of question so the result will be customized according to your needs.
+
+  There are three main ways to generate your Node.js project:
+
+  - Interactive mode - `node-cli g [project name] --interactive`
```

Comment:
Put the commands in front of the descriptions, e.g.
`node-cli g [project name] --interactive` - Interactive mode